apps/template/package.json

```diff
@@ -15,70 +11,48 @@
     "node": ">=20.0.0 <21.0.0",
     "npm": ">=10.0.0 <11.0.0"
   },
-  "devDependencies": {
-    "@babel/core": "^7.20.2",
-    "@babel/preset-env": "^7.20.2",
-    "@babel/preset-react": "^7.18.6",
-    "@cloudoperators/juno-config": "*",
-    "@svgr/core": "^7.0.0",
-    "@svgr/plugin-jsx": "^7.0.0",
-    "@tanstack/react-query": "4.28.0",
-    "@testing-library/dom": "^8.19.0",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^14.4.3",
-    "assert": "^2.0.0",
-    "autoprefixer": "^10.4.2",
-    "babel-jest": "^29.3.1",
-    "babel-plugin-transform-import-meta": "^2.2.0",
-    "jest": "^29.3.1",
-    "jest-environment-jsdom": "^29.3.1",
-    "postcss": "^8.4.21",
-    "postcss-url": "^10.1.3",
-    "react-error-boundary": "^4.0.12",
-    "react-test-renderer": "18.2.0",
-    "sass": "^1.60.0",
-    "shadow-dom-testing-library": "^1.7.1",
-    "tailwindcss": "^3.3.1",
-    "util": "^0.12.4"
-  },
   "scripts": {
-    "build": "NODE_ENV=production node esbuild.config.js",
-    "dev": "NODE_ENV=development node esbuild.config.js --serve --watch",
-    "lint": "eslint .",
-    "test": "jest"
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
```

Comment:
I called this task preview on doop. We might have to agree how to call this task.

Reply:
Yes, lets agree